subm/src/subm.js

```diff
@@ -228,26 +249,27 @@ function makeDiscordBot(guild, authorizedRoles, opts) {
 /**
  * @param {GuildMember} member
  * @param {ReturnType<typeof import('gcs-signed-urls')>} storage
+ * @param { ReturnType<import('./firebaseTool.js').makeFirebaseAdmin> } loadGenAdmin
  */
-function makeTestnetParticipant(member, storage) {
+function makeTestnetParticipant(member, storage, loadGenAdmin) {
   if (!member.user) throw RangeError();
+  const { user } = member;
+  const userID = `${user.username}#${user.discriminator}`;
```

Comment:
Doesn't discord allow people to change their `username`?

Reply:
Perhaps, but a condition of getting testnet points is to not change one's username.